packages/http-server/test/integration/http-server.integration.ts

```diff
@@ -28,16 +31,19 @@ describe('HttpServer (integration)', () => {
       .expect(200);
   });
 
-  it('stops server', async () => {
-    server = new HttpServer(dummyRequestHandler);
-    await server.start();
-    await server.stop();
-    await expect(
-      makeRequest({
-        uri: server.url,
-      }),
-    ).to.be.rejectedWith(/ECONNREFUSED/);
-  });
+  process.env.TRAVIS
+    ? // tslint:disable-next-line:no-unused-expression
+      it.skip
+    : it('stops server', async () => {
+        server = new HttpServer(dummyRequestHandler);
```

Comment:
We could use `{server = new HttpServer(dummyRequestHandler, {host: '127.0.0.1'});` to avoid skip.